packages/react/index.js

```diff
@@ -30,6 +30,8 @@ module.exports = {
     'react/jsx-props-no-spreading': 0,
     'react/no-unused-prop-types': 0,
     'react/require-default-props': 0,
+    'react/prop-types': 0,
+    'no-use-before-define': 0,
```

Comment:
Leider zu spät gesehen, aber die bitte nicht global mit aufnehmen. Also ich weiss nicht, wie es in anderen Projekten aussieht, aber eigentlich sollte die nur verwendet werden, wenn nötig.

Reply:
Wir haben die no-use-before-define im CC wie auch im js-Frontend global abgeschaltet

Reply:
Huch okay, dann hab ich nichts gesagt!